addons/l10n_mt/models/template_mt.py

```diff
@@ -0,0 +1,33 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo import models
+from odoo.addons.account.models.chart_template import template
+
+
+class AccountChartTemplate(models.AbstractModel):
+    _inherit = 'account.chart.template'
+
+    @template('mt')
+    def _get_mt_template_data(self):
+        return {
+            'code_digits': '6',
+            'property_account_receivable_id': 'mt_2050',
+            'property_account_payable_id': 'mt_3100',
+            'property_account_expense_categ_id': 'mt_5500',
```

Comment:
"Opening stock" feels weird for a default account :thinking:

Reply:
Maybe putting the mt_5550 is better